### ISSUE TITLE
On large svn repo prompt with svn status enabled can be slow

### ISSUE THREAD

**Issue by Mikaz-fr** [cmderdev/cmder#2702]
Similarly to #2548 and #1566, on large svn repo fetching the status delay slightly the command prompt display.

I recently updated to Cmder 1.3.19 and was thrilled to see that git status can now be fetched asynchronously using recent clink feature (integrated via #2556). 
Is there any plan to provide the same for svn ?

It's good now we can disable in the option the svn status, but having it asynchronously would be even better.

**Comment by chrisant996** [comment]
There wasn't any plan to do so, because no one had identified it as being a problem that needed solving.  🙃

It's not hard to do.  Someone could make the Lua script tweaks and submit a PR.

(I'll see about extending [clink-flex-prompt](https://github.com/chrisant996/clink-flex-prompt) accordingly.  Someone could port from there to Cmder.  Or if I find myself with extra time, then maybe at some point I can also provide a patch for Cmder, but that's not a priority for me.)

**Comment by Mikaz-fr** [comment]
I've made an attempt to solve this for svn similarly to how it's done for git. I've open pull request #2703.
Please let me know if you would have done that differently.

**Comment by chrisant996** [comment]
The changes look good to me!

**Comment by Mikaz-fr** [comment]
Awesome. Feel free to close once PR is merged. Will get the next release with this feature :)

**Comment by Mikaz-fr** [comment]
This change is now part of release https://github.com/cmderdev/cmder/releases/tag/v1.3.20.
This issue can be closed.
